templates/taxi/manufacturer_list.html

```diff
@@ -23,6 +25,12 @@
               <td>
                   {{ manufacturer.country }}
               </td>
+              <td>
+                <div class="btn-group">
+                  <a class="btn btn-sm btn-outline-secondary" href="{% url "taxi:manufacturer-update"ff manufacturer.id %}">Update</a>
```

Comment:
There is a typo in the `manufacturer-update` URL. It should be `{% url 'taxi:manufacturer-update' manufacturer.id %}`. Also, consider using single quotes to avoid conflicts with HTML attributes.